### ISSUE TITLE
A way to dynamically (programmatically) suspend and resume inbound XCM channels at bridge hub

### ISSUE THREAD

**Issue by svyatonik** [paritytech/polkadot-sdk#1021]
This idea was born here: https://github.com/paritytech/parity-bridges-common/pull/2213#discussion_r1252839684

TLDR: we want the chain that "owns" the bridge endpoint to respect bridge limits and rules. Those limits and rules shall guarantee that this chain uses some kind of rate limiter + there is a running relayer for the dedicated messages lane. When we see that the "owner" violates those rules, we have following options (at the bridge hub):
1) simply start dropping all inbound XCM messages until owner fixes the issue. This is not good, because messages may have bundled assets and those assets will be lost;
2) stop processing inbound XCM messages until owner fixes the issue.

Bridges collective mind has selected the 2nd option. There are couple of options on how to implement that:
1) (this option is assumed by the current implementation of paritytech/parity-bridges-common#2213) add a way to exclude channel from the "service" procedure. For example current `cumulus-pallet-xcmp-queue` has the `service_xcmp_queue` where we can check if the channel is artificially suspended by the bridge (or someone else) and simply stop processing messages. XCM queues have the "suspend" signal that will be sent to the "owner" chain soon after we stop handling inbound messages (when there are more than limit messages in the queue). After this signal is received, the messages will start piling up at the "owner" chain outbound queue instead, which is ideal for us;
2) suggested by @bkontur here: https://github.com/paritytech/parity-bridges-common/pull/2213#discussion_r1258384084. It is to physically close the channel, potentially using XCM instructions (unimplemented). If we'll choose this way, we'll need to adjust current implementation of paritytech/parity-bridges-common#2213 (see "TODOs" referencing this issue in the code).
